rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -210,11 +211,7 @@ rmw_create_node(
     context->impl,
     "expected initialized context",
     return nullptr);
-  RMW_CHECK_FOR_NULL_WITH_MSG(
-    context->impl->enclave,
-    "expected initialized enclave",
-    return nullptr);
```

Comment:
I don't have a good idea of whether or not this check should be in place.  What was your reasoning for removing it?

Reply:
Ah I now store the `enclave` as an `std::string` within `rmw_context_impl_s` so we don't need to check for nullptr. Maybe I can update to check if the string is empty?

Reply:
Oh, I see.  Right, that makes sense.

I kind of feel like an empty enclave is allowed (when not using security).  In that case, I think what you have here is fine.